### ISSUE TITLE
Custom Stack Size for individual foods

### ISSUE THREAD

**Issue by aziel13** [progwml6/HungerOverhaul#144]
It would be nice if the custom food json would also allow for setting stack size for specific items. 

I realized this as I was trying to trade rotten flesh to a villager who was asking for 36 flesh and flesh only stacks up to 16. I tried changing the food value or blacklisting it, but neither of those seem to allow for the standard vanilla stack size.

**Comment by Chemox** [comment]
I ran into this issue as well.
Changing the villagers trades would be an option but that means that its less balanced and easier to get emeralds.

**Comment by Akaitatsu** [comment]
Yes, please. At least make it so the maximum stack size is still 64 and doesn't affect trade items. I use HO with Spice of Life, so being able to stack food to 64 really doesn't make a difference.
